sql_metadb/derived_tables/holdings_tags.sql

```diff
@@ -0,0 +1,27 @@
+-- This derived table extracts tags from the holdings record.
+DROP TABLE IF EXISTS holdings_tags;
+
+CREATE TABLE holdings_tags AS 
+SELECT 
+    hd.instance_id,
+    hd.id AS holding_id,
```

Comment:
For MM, we still want to use holdings. This is how we use it in our other queries.